### PR TITLE
Fix portable pdb debugging

### LINKED ISSUE #1491: Visual Studio Preview 4 --- FSharp debugging with portable pdbs doesn't work
Create a new solution, 
Edit project file to set debugtype to portable pdb.  set a break point.
Debug
Observe that it fails to stop at the breakpoint.

### PR DESCRIPTION
**Pull request by KevinRansom**
Fixes #1491 

https://github.com/Microsoft/visualfsharp/issues/1491

Portable PDB didn't work because the id written to the pdb file (timestamp+MVID) did not match those written to the Debug Record of the PE file.

### CODE REVIEW

File: src/absil/ilwritepdb.fs [hunk 1]
```diff
@@ -95,31 +95,34 @@ type idd =
...
 
 let magicNumber = 0x53445352L
-let pdbGetDebugInfo (mvid:byte[]) (filepath:string) = 
+let pdbGetDebugInfo (mvid:byte[]) (timestamp:int32) (filepath:string) = 
     let iddDataBuffer = 
         let path = (System.Text.Encoding.UTF8.GetBytes filepath)
         let buffer = Array.zeroCreate (sizeof<int32> + mvid.Length + sizeof<int32> + path.Length + 1)
 
-        let offset, size = 0, sizeof<int32>                                     // Magic Number RSDS dword: 0x53445352L
+        let struct (offset, size) = struct(0, sizeof<int32>)                    // Magic Number RSDS dword: 0x53445352L
```
Comment (dsyme) [inline src/absil/ilwritepdb.fs]: Why use struct tuples here?  This isn't perf critical code.

**Comment by KevinRansom** [comment]
Why not use struct tuples?

**Comment by dsyme** [comment]
Given the inevitable syntactic overhead (even if eventually reduced somewhat), they should only be used if there's a specific reason to use them (i.e. perf, interop). 

In the compiler we could introduce them after seeing profiling data indicating high allocation rates, or tests indicating perf improvement. That's what they are for, at least for F#.

Some uses for dogfooding are ok too :)

**Comment by enricosada** [comment]
nice! i'll try in vscode too with new compiler

**Comment by KevinRansom** [comment]
It’s still not great ….

**Pull request merged**